src/classes/LineItemTriggerHandler.cls

```diff
@@ -42,7 +46,7 @@ public class LineItemTriggerHandler {
 		for (Order__c o : orders) {
 			Decimal orderAmount = orderToAmountMap.get(o.Id); 
 			o.Purchased__c = (orderAmount!=null) ? orderAmount : 0;
-//			update o;
+			update o;
```

Comment:
DML statement `update o;` has no exception handling. Surround with a `try/catch` block to handle possible exceptions.